### PR TITLE
Linting lesson: Fix broken link for article 1

### PR DESCRIPTION
**Pull request by sikai00**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

The current link leads to an error 404.

**2. This PR:**

This is solved by replacing the link with one found in Wayback Machine. This is suitable as there had been links in many previous lessons leading to an archived version of a webpage using the Wayback Machine.

**3. Additional Information:**

### CODE REVIEW

File: javascript/javascript_in_the_real_world/linting.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Code style is important! Having a consistent set of style rules for things such
 
 The style guides we mentioned above are full of really helpful advice for formatting, organizing and composing your code. But there are a _lot_ of rules - it can be difficult to internalize them all. **Linters** are tools that will scan your code with a set of style rules and will report any errors to you that they find. In some cases, they can even auto-fix the errors! The following articles explain in more detail the benefits of using a linter while you code.
 
-1. [This article](http://mikecavaliere.com/javascript-linting-what-developers-need-to-know/) gets right to the point... start here!
+1. [This article](https://web.archive.org/web/20220113100650/https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/) gets right to the point... start here!
```
Comment (01zulfi) [inline javascript/javascript_in_the_real_world/linting.md:17]: I don't think we'll need to use the archive link, as I was able to find the article on the same website. Looks like they moved their articles to `/posts`: https://mikecavaliere.com/posts/javascript-linting-what-developers-need-to-know

**Review by 01zulfi — CHANGES_REQUESTED** [review]

File: javascript/javascript_in_the_real_world/linting.md (hunk 1 above)
Comment (sikai00) [reply javascript/javascript_in_the_real_world/linting.md:17]: Hey that's great! Didn't realise that was the case, thanks!

Comment (01zulfi) [reply javascript/javascript_in_the_real_world/linting.md:17]: Would you be willing to make that change?

Comment (sikai00) [reply javascript/javascript_in_the_real_world/linting.md]: I've made the changes! Sorry it took me awhile to figure this out

Comment (01zulfi) [reply javascript/javascript_in_the_real_world/linting.md]: No worries, thanks for contributing :rocket:

**Review by 01zulfi — APPROVED** [review]

File: javascript/javascript_in_the_real_world/linting.md (hunk 1 above)
Comment (sikai00) [reply javascript/javascript_in_the_real_world/linting.md]: Hey, my apologies, I didn't realise that the links under Knowledge check linked to the same webpage and didn't update them. Should I open a new pull request?

Comment (01zulfi) [reply javascript/javascript_in_the_real_world/linting.md]: Thanks for checking. Yes, opening a new pull request will be ideal.

Comment (sikai00) [reply javascript/javascript_in_the_real_world/linting.md]: Okay, I've opened a new pull request for this matter

**Pull request merged**